### PR TITLE
Ex03

### PR DESCRIPTION
**Pull request by mikkonator97**

### CODE REVIEW

**Comment by mikkonator97** [comment]
The bubble_sort function remains the same, since the solution was not yet available for me to check out. Still works fine though 👍

File: src/mikko_rekstad_ex/ex03_project/test_sorting.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+    data = [3, 2, 1]
+    sorted_data = bubble_sort(data)
+
+    Now sorted_data shall be a different object than data,
+    not just another name for the same object.
+    """
+    data = [3, 2, 1]
+    sorted2 = bubble_sort(data)
+    assert id(data) is not id(sorted2)
```
Comment (huynhngoc) [inline src/mikko_rekstad_ex/ex03_project/test_sorting.py]: `assert data is not sorted2`

File: src/mikko_rekstad_ex/ex03_project/test_sorting.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    data = [3, 2, 1]
+    sorted_data = bubble_sort(data)
+
+    Now data shall still contain [3, 2, 1].
+    """
+    data = [3, 2, 1]
+    original_data = [3, 2, 1]
+    sorted_data = bubble_sort(data)
+    for number in data:
+        assert original_data.count(number) == sorted_data.count(number)
```
Comment (huynhngoc) [inline src/mikko_rekstad_ex/ex03_project/test_sorting.py]: `assert data == original_data`

Comment (mikkonator97) [reply src/mikko_rekstad_ex/ex03_project/test_sorting.py]: My bad, will name it correctly next time

**Pull request merged**